### PR TITLE
Upgrade code for framework namespaces

### PR DESCRIPTION
**Pull request by tractorcow**
Merge with https://github.com/silverstripe/silverstripe-framework/pull/5921

Remove `@package / @subpackage`
Fix inifinite loop in VirtualPage_Controller

### CODE REVIEW

File: code/Model/SiteTree.php [hunk 1]
```diff
@@ -2036,12 +2033,6 @@ public function getCMSFields() {
 				. "</p>"), "Title");
 		}
 
-		// Backwards compat: Rewrite nested "Content" tabs to toplevel
```
Comment (chillu) [inline code/Model/SiteTree.php]: We haven't thrown deprecation notices on this, so might catch some devs out - but we've mentioned it in the 3.0.0 changelog: https://github.com/silverstripe/silverstripe-framework/blob/master/docs/en/04_Changelogs/3.0.0.md#changed-tab-paths-in-sitetree-getcmsfields-tab-paths

File: code/Model/SiteTreeLinkTracking_Parser.php [hunk 2]
```diff
@@ -91,7 +92,7 @@ public function process(SS_HTMLValue $htmlValue)
 					'Target' => $matches['id'],
 					'Anchor' => null,
 					'DOMReference' => $link,
-					'Broken' => !DataObject::get_by_id('File', $matches['id'])
+					'Broken' => !DataObject::get_by_id('SilverStripe\\Assets\\File', $matches['id'])
```
Comment (chillu) [inline code/Model/SiteTreeLinkTracking_Parser.php]: We could take this as an opportunity to rewrite this all to the less verbose `File::get()->byID()` ... but then again, that's annoyingly not cached in the same way as `get_by_id()`

File: code/Model/SiteTree.php (hunk 1 above)
Comment (tractorcow) [inline code/Model/SiteTree.php]: Yes we have actually. See FieldList::rewriteTabPath in 3.x

File: code/Model/SiteTreeLinkTracking_Parser.php (hunk 2 above)
Comment (tractorcow) [inline code/Model/SiteTreeLinkTracking_Parser.php]: Yeah that's the only reason. =(

**Pull request merged**